### PR TITLE
Add license type to VMC

### PR DESCRIPTION
**Pull request by HenryZhu123**
Sample output:

Screen Shot 2019-07-08 at 11 51 01 AM

### CODE REVIEW

File: src/frontend/org/voltdb/RealVoltDB.java [hunk 1]
```diff
@@ -1219,6 +1219,7 @@ public void initialize(Configuration config) {
                 jo.put("commandlogging", m_licenseApi.isCommandLoggingAllowed());
                 jo.put("wanreplication", m_licenseApi.isDrReplicationAllowed());
                 jo.put("expiration", sdf.format(m_licenseApi.expires().getTime()));
+                jo.put("type", m_licenseApi.getLicenseType());
```
Comment (Chaozz) [inline src/frontend/org/voltdb/RealVoltDB.java]: This change will also add a new item in the result of `@SystemInformation "overview"`. We should let Andrew @ajgent  know about the change so we can update the doc.

File: src/frontend/org/voltdb/dbmonitor/js/voltdb.ui.js [hunk 2]
```diff
@@ -1018,9 +1018,10 @@ var loadPage = function (serverName, portid) {
                 $(".licenseInfo").show();
                 $("#tdLicenseInfo").hide();
                 $("#tdLicenseInfo").css("display", "none");
+                $("#tdLicenseType").html(licInfo.type);
```
Comment (Chaozz) [inline src/frontend/org/voltdb/dbmonitor/js/voltdb.ui.js]: Did you try it on the community edition? I guess the license type won't show up in a community server.

Comment (Chaozz) [reply src/frontend/org/voltdb/dbmonitor/js/voltdb.ui.js]: Ok, I think it works in the community edition after reading the code.

Comment (HenryZhu123) [reply src/frontend/org/voltdb/dbmonitor/js/voltdb.ui.js]: If it's community edition it displays "Community Edition" only, none of the other information. There was a pre-existing code for handling community edition, essentially an if else a few lines above this that checks if any information about the license was passed, and if it was does the above, if not just displays "Community Edition".

Comment (Chaozz) [reply src/frontend/org/voltdb/dbmonitor/js/voltdb.ui.js]: Got it. Actually, you can just hard code `$("#tdLicenseType").html("enterprise edition");` without processing the LicenseType info. When we have license info, it is guaranteed to be an enterprise.

Comment (HenryZhu123) [reply src/frontend/org/voltdb/dbmonitor/js/voltdb.ui.js]: Is it? I recall Andrew @ajgent mentioning in an email a while back that there were "Enterprise Edition", "Pro Edition", and "AWS Marketplace" for license type.

Comment (Chaozz) [reply src/frontend/org/voltdb/dbmonitor/js/voltdb.ui.js]: My bad, I only heard about Enterprise and community. if there are more types, then you should keep your change

File: src/frontend/org/voltdb/RealVoltDB.java (hunk 1 above)
Comment (HenryZhu123) [reply src/frontend/org/voltdb/RealVoltDB.java]: Andrew has been notified and approved the change to SystemInformation OVERVIEW.

**Pull request merged**